install.ps1

```diff
@@ -401,6 +425,7 @@ function Write-Env {
             [Microsoft.Win32.RegistryValueKind]::String
         }
         $EnvRegisterKey.SetValue($name, $val, $RegistryValueKind)
+        Publish-Env
```

Comment:
Should this be placed to two lines below? I suppose that `DeleteValue` is also an action involving env modification.

Reply:
I think it should not be a big problem not to publish after deletion, but it will not be wrong if it is applied.